### ISSUE TITLE
API failed to response

### ISSUE THREAD

**Issue by x0xa** [ethereum-mining/ethminer#2075]
Hi all!

After 0.17 release API not works for any requests with PHP code

```
$fp = fsockopen("127.0.0.1", 3333, $errno, $errstr, 5);
if (!$fp) {
    echo "$errstr ($errno)\n";
} else {
	echo "connection established\n";
	
    $query = '{"id":0,"jsonrpc":"2.0","method":"miner_ping"}';
    	
    fwrite($fp, $query);
	fwrite($fp, "\n");
	
    while (!feof($fp)) {
        echo fgets($fp, 128);
    }
	
    fclose($fp);
}
```

0.17 console log:
connection established
{"id":0,"jsonrpc":"2.0","result":"pong"}

0.18 and above:
connection established

and that's all .....

please, fix it, dudes!!!
Thanks all!!!

**Comment by danielocdh** [comment]
This was fixed in #1968, the problem is that they haven't released a new release since then(or any 0.19 release), for windows you can find a compiled version in main page (https://github.com/ethereum-mining/ethminer) in the AppVeyor link. I used that but...the API response of the new version is different than the API in 0.17.1, so if you are using a third party tool to monitor, it is unlikely they update it because there is no official 0.19 release

**Comment by x0xa** [comment]
Thank you very much!

**Comment by AndreaLanfranchi** [comment]
Closed by #1968
